elpis/engines/hftransformers/objects/model.py

```diff
@@ -18,6 +18,7 @@
 from sklearn.model_selection import train_test_split
 import torch
```

Comment:
Expect this to add significantly (at least 1-2GB) to the image size.

Reply:
11.35GB now. Watch me try and push this over my phone's mighty 3G connection!